fili-core/src/test/java/com/yahoo/bard/webservice/web/endpoints/JobsEndpointResources.java

```diff
@@ -49,7 +49,7 @@ public static ApiJobStore getApiJobStore() {
         fieldValueMap1.put(DefaultJobField.JOB_TICKET, "ticket1");
         fieldValueMap1.put(DefaultJobField.DATE_CREATED, "2016-01-01");
         fieldValueMap1.put(DefaultJobField.DATE_UPDATED, "2016-01-01");
-        fieldValueMap1.put(DefaultJobField.QUERY, "https://localhost:9998/v1/data/QUERY");
+        fieldValueMap1.put(DefaultJobField.QUERY, "https://localhost:PORT/v1/data/QUERY");
```

Comment:
The only places this refactor gets weird is when `JobsEndpointResources` is used (i.e. `JobsServletSpec.groovy`).

This is because we need to build the binder (including `JobsEndpointResources`) to start jersey, but we need to start jersey to know the port and I couldn't think of a way to inject the port.